sql/moz-fx-data-shared-prod/org_mozilla_ios_firefox_derived/review_checker_clients_v1/query.sql

```diff
@@ -0,0 +1,108 @@
+WITH shopping_metrics AS (
+  SELECT
+    client_info.client_id AS client_id,
+    DATE(submission_timestamp) AS submission_date,
+    SUM(
+      CASE
+        WHEN metrics.counter.shopping_product_page_visits IS NOT NULL
+          THEN COALESCE(CAST(metrics.counter.shopping_product_page_visits AS INT64), 0)
+        ELSE 0
+      END
+    ) AS shopping_product_page_visits,
+    CASE
+      WHEN metrics.boolean.shopping_settings_user_has_onboarded = TRUE
+        AND metrics.boolean.shopping_settings_component_opted_out = FALSE
+        THEN 1
+      ELSE 0
+    END AS is_opt_in,
+    CASE
+      WHEN metrics.boolean.shopping_settings_component_opted_out = TRUE
+        THEN 1
+      ELSE 0
+    END AS is_opt_out,
+    CASE
+      WHEN metrics.boolean.shopping_settings_user_has_onboarded = TRUE
+        THEN 1
+      ELSE 0
+    END AS is_onboarded,
+    CASE
+      WHEN metrics.boolean.shopping_settings_nimbus_disabled_shopping = TRUE
+        THEN 1
+      ELSE 0
+    END AS is_nimbus_disabled,
+    normalized_channel,
+    normalized_country_code,
+    sample_id,
+    ANY_VALUE(ping_info.experiments) AS experiments
+  FROM
+    `moz-fx-data-shared-prod.org_mozilla_ios_firefox_stable.metrics_v1` AS m
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+  GROUP BY
+    client_id,
+    submission_date,
+    is_opt_in,
+    is_opt_out,
+    normalized_channel,
+    normalized_country_code,
+    sample_id,
+    is_onboarded,
+    is_nimbus_disabled
+),
+active AS (
+  SELECT
+    client_info.client_id,
+    DATE(submission_timestamp) AS submission_date,
+    COALESCE(SUM(metrics.timespan.glean_baseline_duration.value), 0) / 3600.0 AS active_hours
+  FROM
+    `moz-fx-data-shared-prod.firefox_ios.baseline`
+  WHERE
+    DATE(submission_timestamp) > DATE_SUB(CURRENT_DATE(), INTERVAL 2 DAY)
```

Comment:
Same question as above

```suggestion
    DATE(submission_timestamp) > DATE_SUB(@submission_date, INTERVAL 2 DAY)
```